src/main/java/io/odpf/firehose/sink/objectstorage/ObjectStorageSink.java

```diff
@@ -58,4 +78,13 @@ public boolean canManageOffsets() {
         return true;
     }
 
+    @Override
+    public void addOffsets(Object batch, List<Message> messageList) {
```

Comment:
let's call this 
addOffsets(Object key, List<Message> messageList);

Reply:
renamed